enhancements/network/user-defined-network-segmentation.md

```diff
@@ -836,6 +848,11 @@ Having the prefix avoids conflicting with existing NADs who has the same `metada
 For example:
 Given the CR meta.name is `db-network`,the NAD metadata.name will be `cluster.udn.db-network`.
 
+If a VRF name is provided in the cluster-scope CRD instance, it will be set to an homonimous field in the NAD NetConf. If the 
+provided VRF name is already used in any other cluster-scope CRD instance, a failed status should be reported for the cluster-scope 
```

Comment:
Sounds like it"ll require additional caching, from what I know its something that should be avoided.

Could it be validated at the API level? using some CEL rules?

Reply:
I don't think so.